cogs/birthday/birthday.py

```diff
@@ -118,7 +118,7 @@ async def test(self, ctx):
         for msg in CANNED_MESSAGES:
             await ctx.send(msg.format(ctx.author.mention))
 
-    @_birthday.command(name="add")
+    @_birthday.command(name="add", aliases=["set"])
```

Comment:
Added a `set` alias for the `add` command to make it easier on mods/admins since that was the command before the module was refactored.